test/BaselineOutput/SingleRelease/SavePipe/TestParquetNull-Data.txt

```diff
@@ -0,0 +1,9 @@
+#@ TextLoader{
```

Comment:
I wish we had a mechanism in place so we didn't have to duplicate baseline files between Debug and Release when the files were identical.

Reply:
Me too but unfortunately beyond the scope of this change. Lets open an issue if there isn't one open. 

---
In reply to: [213393285](https://github.com/dotnet/machinelearning/pull/757#discussion_r213393285) [](ancestors = 213393285)

Reply:
Can you open one?

Reply:
Done. #758